plugins/by-name/mini/default.nix

```diff
@@ -5,6 +5,8 @@
 lib.nixvim.neovim-plugin.mkNeovimPlugin {
   name = "mini";
   packPathName = "mini.nvim";
+  # Just want it before most other plugins for the icons provider.
+  configLocation = lib.mkBefore "extraConfigLua";
```

Comment:
I'm minded to suggest we avoid using default prio numbers internally, that way end users can reliably go "before" us if we consistently use a prio after `(mkBefore null).priority`, for example

IIRC, before is 500, default is 1000, and after is 1500? So perhaps we should aim to be in the 7xx & 8xx range?